libs/SpatialAlignment-Unity/Assets/SpatialAlignment.Unity/Placeholder.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
```

Comment:
nit: this file may serve no purpose right now, but you should add the microsoft license still to the top

Reply:
Agreed.